doc-surrealdb_versioned_docs/version-2.x/cli/ml/export.mdx

```diff
@@ -142,6 +157,7 @@ Options:
   -e, --endpoint <ENDPOINT>      Remote database server url to connect to [default: ws://localhost:8000] [aliases: conn]
   -u, --username <USERNAME>      Database authentication username to use when connecting [env: SURREAL_USER=] [aliases: user]
   -p, --password <PASSWORD>      Database authentication password to use when connecting [env: SURREAL_PASS=] [aliases: pass]
+    -t, --token <TOKEN>            Authentication token in JWT format to use when connecting
```

Comment:
Is perhaps the environment variable missing here?

Reply:
Pushing those in the next commit